src/Generators/MigrationGenerator.php

```diff
@@ -100,6 +102,6 @@ protected function getTableRow($fieldName, $typeName): string
             return $this->getNonRequiredLine($fieldName, $typeName);
         }
 
-        throw new Exception('Unknown fieldType in MigrationGenerator');
+        throw new UnknownFieldTypeException("Unknown field type {$typeName} in MigrationGenerator.");
```

Comment:
let's move message generation into the exception class

```suggestion
        throw new UnknownFieldTypeException($typeName, 'MigrationGenerator');
```